app/workers/organizations/delete_worker.rb

```diff
@@ -0,0 +1,44 @@
+module Organizations
+  class DeleteWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
+
+    # operator_id - the user who deletes the organization
+    def perform(organization_id, operator_id)
+      org = Organization.find_by(id: organization_id)
+      return unless org
```

Comment:
Non blocking comment/question:
I know this is a pattern we use often - is a job to delete an org that doesn't exist getting started log-worthy? 
Right now we just "succeed" quietly.

Reply:
Welcome to Michael's Pet Peeves, episode 42. 😂 Joking aside, this is something we've been discussing for a long time but haven't changed yet. IMO the real crux of the issue is that we set up our jobs with retries, but don't actually want or need them in many cases (like here when the org doesn't exist).

Reply:
It seems to be log-worthy. Does it make sense to create an issue/rfc for this?

Reply:
An RFC for error handling in workers would definitely be appreciated. If you have time for it and feel like writing one that'd be great, otherwise it's on my list but it may take a bit before I get there.